ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -117,6 +130,93 @@ public ReactScrollView(ReactContext context, @Nullable FpsListener fpsListener)
     mScroller = getOverScrollerFromParent();
     setOnHierarchyChangeListener(this);
     setScrollBarStyle(SCROLLBARS_OUTSIDE_OVERLAY);
+
+    ViewCompat.setAccessibilityDelegate(
+      this,
+      new AccessibilityDelegateCompat() {
+
+        @Override
+        public void onInitializeAccessibilityEvent(View host, AccessibilityEvent event) {
```

Comment:
@blavalla Would you recommend putting `onInitializeAccessibilityEvent` in a separate class which would handle AccessibilityDelegate for Horizontal or Vertical ScrollView?